docs/docsite/rst/user_guide/complex_data_manipulation.rst

```diff
@@ -0,0 +1,71 @@
+Complex data manipulation
+#########################
+
+In many cases you need to do some complex operation with your variables, while Ansible is not recommended as a data processing/manipulation tool, you can use the existing Jinja2 templating in conjunction with the many added ansible filters, lookups and tests to do some very complex transformations.
+
+A quick definition of each type of plugin before we go forward
+  - lookups: Mainly used to query 'external data', in Ansible these were the primary part of loops using the `with_<lookup>` construct, but they can be used independantly to return data for processing. They normally returna list due to their primary function in loops as mentioned previouslly. Used with the ``lookup`` or ``query`` Jinja2 operators.
+  - filters: used to change/transform data, used mainly with the ``|`` Jinja2 operator.
```

Comment:
s/mainly //

Reply:
map/select as the 'other usage', so they are not exclusively used with | or is